sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aad/properties/AADResourceServerProperties.java

```diff
@@ -38,10 +38,11 @@
      * Configure which claim in access token be returned in AuthenticatedPrincipal#getName. Default value is "sub".
      */
     private String principalClaimName;
+
     /**
      * Configure which claim will be used to build GrantedAuthority, and prefix of the GrantedAuthority's string value.
      * Default value is: "scp" -> "SCOPE_", "roles" -> "APPROLE_".
-     * @see org.springframework.security.core.GrantedAuthority
+     * @see org.springframework.security.core.GrantedAuthority .
```

Comment:
We should remove this @see xxxx, the @see won't do much help in the metadata.


https://docs.spring.io/spring-boot/docs/current/reference/html/configuration-metadata.html#configuration-metadata.annotation-processor
"
You should only use plain text with @ConfigurationProperties field Javadoc, since they are not processed before being added to the JSON.
"

Reply:
Deleted.